tslint.json

```diff
@@ -80,6 +80,7 @@
         ],
         "ordered-imports": false,
         "prefer-const": false,
+        "prefer-for-of": false,
```

Comment:
By default tslint want you to use `for(let v in vs)` instead of a basic for loop. It's significantly less performant. Disabling the warning.